react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -26,19 +35,200 @@ const DEFAULT_BUTTON_CONFIGURATION = {
  */
 class InfoDialogButton extends Component {
     /**
+     * {@code InfoDialogButton} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Whether or not {@code InfoDialog} should be displayed.
+         */
+        _showDialog: PropTypes.bool,
+
+        /**
+         * Whether or not the toolbars, in which this component exists, are
+         * visible.
+         */
+        _visible: PropTypes.bool,
```

Comment:
Maybe it would improve readability if we rename this variable to _toolboxVisible.